src/Sylius/Bundle/UserBundle/Controller/SecurityController.php

```diff
@@ -56,12 +59,13 @@ public function logoutAction(Request $request)
      * Renders the login template with the given parameters. Overwrite this function in
      * an extended controller to provide additional data for the login template.
      *
-     * @param array $data
+     * @param string $view The view name
+     * @param array  $data
      *
      * @return \Symfony\Component\HttpFoundation\Response
      */
-    protected function renderLogin(array $data)
+    protected function renderLogin($view, array $data)
```

Comment:
This should be named ``template``, right? It contains template name. View is confusing, I thought about View object.